app/models/miq_report/generator.rb

```diff
@@ -88,6 +88,34 @@ def get_include
     include_as_hash(include.presence || invent_report_includes)
   end
 
+  def polymorphic_includes
+    @polymorphic_includes ||= begin
+      polymorphic_rels = []
+      top_level_rels   = []
```

Comment:
could we just `+` the two `Array()` calls from `top_level_rels +=` and not do a `+=`?

Reply:
Not sure what you are suggesting here, but I will try and slim it down.  I realize it is pretty ugly.

Suggestions welcome.

Reply:
Did as you suggested (I think).  Let me know if you were thinking of something else.